### ISSUE TITLE
Add a setting to help unblock elements

### ISSUE THREAD

**Issue by Dormann** [uBlockOrigin/uBlock-issues#192]
### Prerequisites

- [x] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

Greetings,

I've started using hard mode with dynamics filtering.
And I've also read the warnings about pages being often broken. ^^

It's just that when I browse a page, sometimes, it's really hard to guess what kind of element was block and which rule to lift in order to have this element again.
Would it be possible to have a setting for a mode in which the elements which are blocked are replaced by a frame, at the same place, telling what kind of object it is supposed to be, from which website it comes and, maybe, a possibility for suggesting the least permissive action to do to allow this element (if you need to noop or allow a rule).

Best regards.

### Your environment

* uBlock Origin version: 1.16.20
* Browser Name and version: Firefox 61.0.2
* Operating System and version: Windows 7

**Comment by uBlock-user** [comment]
Sounds like https://github.com/gorhill/uBlock/issues/3619

@Dormann Is that what you're suggesting ?

**Comment by Dormann** [comment]
@gwarser : no, DOM Inspector can help with creating filter rules about allowed elements. 

@uBlock-user  : ah, yes, from what I've read, it would be very similar to (if not exactly)  what i meant.
The description about Granular placeholder seems to fit.
Sorry for the duplicate, I though that I had cheched every keywords.

**Comment by uBlock-user** [comment]
Duplicate of https://github.com/gorhill/uBlock/issues/2688
